assets/_sass/base/_variables.scss

```diff
@@ -1,14 +1,13 @@
 // 18F Fonts
 $raleway: Raleway,'Helvetica Neue',Helvetica,Arial,sans-serif;
 $open-sans: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
-$brand: '18F Nimbus','Helvetica Neue',Helvetica,sans-serif;
-$nimbus: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
+$brand: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
```

Comment:
i think the extra files changed are from the merge, instead of a rebase which would make the history cleaner.